.github/workflows/ci.yml

```diff
@@ -126,8 +126,8 @@ jobs:
         run: ./v build-vbinaries
       - name: Test v tutorials
         run: |
-          cd tutorials/code/blog
-          ../../../v .
+          cd tutorials/building_a_simple_web_blog_with_vweb/code/blog
+          ../../../../v .
```

Comment:
I think `./v tutorials/building_a_simple_web_blog_with_vweb/code/blog`
or
```
cd tutorials
../v building_a_simple_web_blog_with_vweb/code/blog
```
is somewhat cleaner

Reply:
thanks ^^, it's fixed now, I used the form `./v tutorials/building_a_simple_web_blog_with_vweb/code/blog`